cmake/Modules/FindPythonExtra.cmake

```diff
@@ -17,211 +17,85 @@
 #
 # CMake module for providing extra information about the Python interpreter.
 #
-# Output variables:
+# Output Targets:
+# - Python3::Interpreter - an executable target that will invoke a version of Python
+#     matching CMAKE_BUILD_TYPE on platforms where that matters
 #
+# Output variables:
 # - PythonExtra_FOUND: True if a Python executable was found
-# - PythonExtra_EXTENSION_SUFFIX: The suffix for a Python extension, according
-#    to PEP-3149: https://www.python.org/dev/peps/pep-3149/
-# - PythonExtra_EXTENSION_EXTENSION: The extension for a Python extension. On
-#    Linux and Mac OSX equals to ".so", on Windows to ".pyd"
-# - PythonExtra_INCLUDE_DIRS: The paths to the directories where the Python
-#    headers are installed.
-# - PythonExtra_LIBRARIES: The paths to the Python libraries.
-# - PYTHON_SOABI: The shared library file name tag according to PEP-3149:
-#    https://www.python.org/dev/peps/pep-3149/
 #
-# Conditional output variables
+# Advanced Output variables
+# - PYTHON_EXECUTABLE: a path to a python interpreter (deprecated; new code should use PythonExtra::Interpreter)
 # - PYTHON_EXECUTABLE_DEBUG: If the CMAKE_BUILD_TYPE is Debug and WIN32 is true
-#    then this will be a path to a debug build of the Python interpreter.
+#    then this will be a path to a debug build of the Python interpreter (deprecated; new code should use PythonExtra::Interpreter)
+# - PythonExtra_POSTFIX: a postfix that downstream consumers can use as the DEBUG_POSTFIX property to a target
 #
 # Example usage:
 #
 #   find_package(python_cmake_module REQUIRED)
 #   find_package(PythonExtra MODULE)
-#   # use PythonExtra_* variables
+#   # use PythonExtra::Interpreter
```

Comment:
Nit, `PythonExtra::Interpreter` -> `Python3::Interpreter`

Reply:
Fixed in 51b769c